mirdata/annotations.py

```diff
@@ -79,6 +79,26 @@ def __repr__(self):
         return repr_str
 
 
+class MultiAnnotator(Annotation):
+    """Multiple annotator class.
+    This class should be used for datasets with multiple annotators (e.g. multiple annotators per track).
+
+    Attributes:
+        annotators (list): list with annotator ids
+        annotations (list): list of annotations (e.g. [annotations.BeatData, annotations.ChordData]
+    """
+
+    def __init__(self, annotators, annotations) -> None:
+        validate_array_like(annotators, list, str, none_allowed=True)
+        validate_array_like(
+            annotations, list, Annotation, check_child=True, none_allowed=True
+        )
+        validate_lengths_equal([annotators, annotations])
+
+        self.annotators = annotators
+        self.annotations = annotations
```

Comment:
maybe too much for this PR, but thought for the future - 

this class could have methods like: 
`.get(annotator_id)`
`.get_best()`  (which uses a dataset-defined heirarchy when initialized saying which annotations are "best")

Reply:
Yeah, plus we could have methods with different ways of combining the annotations, which in some cases make sense! Like minimum vote, or all annotators agree, etc. We can do that in a follow-up PR